haskell/ghc.BUILD

```diff
@@ -25,13 +25,6 @@ cc_library(
     )[0],
 )
 
-# This is needed for Hazel targets.
```

Comment:
Why remove this? I depend on this on a different project. Could you either keep it or replace it with something like

``` python
filegroup(name = "all", srcs = glob(["**"]), visibility = ["//visibility:public"])
```
so that people can still access the header files?

Reply:
Huh, just a revert failure on my part, I guess :)